apps/style/courses.scss

```diff
@@ -0,0 +1,15 @@
+@import "pdf_printing";
+
+@media print {
+  /* we have many components that we never want to include in a print view */
+  .announcement-notification,
+  .course-overview-top-row {
+    display: none;
+  }
+
+  #uitest-version-selector,
+  .teacher-info-box,
+  .uitest-go-to-unit-button {
+    display: none !important;
+  }
```

Comment:
comment thread not blocking this PR:

@madelynkasula since you are looking into the component library and css-in-js stuff, I just wanted to point out that because our current bast practices are to use inline styling, we have to use `!important` (a general no-no / last resort) in order to override styles for printing. I think we have some good momentum around this approach of using print styles to generate PDFs, so it's worth considering this as part of the component library work. that said, if we are willing to "allowlist" print styles for using `!important` then that could be a reasonable compromise.

Reply:
one way to frame this is that if we continue with inline styles as the norm, we should consider making it strictly forbidden to use `!important` on inline styles, so that we retain the freedom to customize appearance when generating PDFs.

Reply:
related: the extent to which inline styles necessitate the use of `!important` in situations like this is one of the main reasons I'd like to see us move away from them

Reply:
thanks for sending this my way -- this is a really good example. i think there are a few ways to avoid situations like this while still using CSS-in-JS, but i haven't looked at printing on its own yet. either way, we'll have to refactor away our current usage of `!important`, which i actually have [a task](https://codedotorg.atlassian.net/browse/STAR-1469) for this sprint (just looking at where/how it's used and how we could approach refactoring)!